content/docs/forms/file-input.mdx

```diff
@@ -0,0 +1,54 @@
+---
+title: React File Input - Flowbite
+description: Get started with the file input component to let the user to upload one or more files from their device storage based on multiple styles and sizes
+---
+
+The `<FileInput>` component can be used to upload one or more files from the device storage of the user available in multiple sizes, styles, and variants and built with the utility-first classes from Tailwind CSS including support for dark mode.
+
+Make sure that you have included Flowbite as a plugin inside your Tailwind CSS project to apply all the necessary styles for the `<FileInput>` component.
+
+To start using the component make sure that you have imported it from Flowbite React:
+
+```jsx
+import { FileInput } from 'flowbite-react';
+```
+
+## File upload example
+
+Get started with a simple `<FileInput>` component to let users upload one single file.
+
+<Example name="fileInput.example" />
```

Comment:
to keep the pattern should we rename to `fileInput.root`? 🤔

Reply:
Yes I will change.